galaxy_importer/loaders.py

```diff
@@ -281,7 +283,29 @@ def _get_readme(self):
 
     def _get_metadata_description(self):
         self.log.info('Getting role description')
-        return ''
+        description = None
+        metadata_file = self._get_metadata_file(self.rel_path)
+
+        if not metadata_file:
+            self.log.warning('No role metadata found')
+            return description
+
+        with open(metadata_file) as fp:
+            role_metadata = yaml.safe_load(fp)
+        try:
+            description = role_metadata['galaxy_info']['description']
+        except KeyError:
+            self.log.warning('No role description found in role metadata')
+        return description
+
+    @staticmethod
+    def _get_metadata_file(rel_path):
+        """Gets path to role metadata file."""
+        for file in ROLE_META_FILES:
```

Comment:
Does galaxy-import need to support py2?  Under py2, 'file' is a built in object, so using it as a var name can sometimes cause odd behavior. 

For py3 only, 'file' is no longer a builtin, so should be fine.

Reply:
just python3